src/main/java/me/totalfreedom/totalfreedommod/command/Command_ov.java

```diff
@@ -15,7 +15,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.IMPOSTOR, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Overlord - control this server in-game", usage = "access", aliases = "ov")
+@CommandParameters(description = "Overlord - control this server in-game", usage = "access", aliases = "overlord")
```

Comment:
Perhaps these should be the other way around. The full filename should be `Command_overlord`, and the alias `ov`.

Reply:
So this was just to fix a bug where the alias and file name were exactly the same. If you think we should change it I am inclined to say we should raise a new issue for the change and put it in a later release.

Reply:
If you fix something that's broken, fix it good then. From a gameplay perspective, it doesn't change much, solely the help pages.

Reply:
The ticket requirements clearly state that the alias should be changed to /overlord from /ov as per the issue https://github.com/TotalFreedom/TotalFreedomMod/issues/1652 - That is what I have done.